index.html

```diff
@@ -1,63 +1,97 @@
+<!-- js script added with dropdown; block titles with layered div column layout to organize boxes; 
+added folders and files to main project folder; all addiitonal css files are imported in the styles.css -->
+<html>
+<head>
+	<title>Introduction to the DOM</title>
+	<link rel="stylesheet" href="./styles.css">
+	<script defer="defer" src="./index.js"></script>
+</head>
+<body>
+	<div class="Block Block__H">
+		<script async src="components/Dropdown/Dropdown.js"></script>
+		<div class="Dropdown">
+			<div class="Dropdown__Title">Dropdown</div>
+			<div class="Dropdown__List">
+					<a href="https://www.lambdaschool.com" alt="Lambda School Link">Lambda</a>
+					<a href="https://www.google.com" alt="Link to Google">Google</a>
+					<a href="https://developer.mozilla.org/en-US/" alt="Link to MDN">MDN</a>
 			</div>
 		</div>
+		<h3>Header</h3>
 	</div>
-	<div class="Tabs__item" data-tab="2">
-		<div class="Box">
-			<div class="Box__title">Box # 2</div>
-			<div class="Box__description">There was an old man of St. Bees,
-				Who was stung in the arm by a wasp,
-				<br>
-				When asked, "Does it hurt?"
-				<br>
-				He replied, "No, it doesn't,
-				<br>
-				I'm so glad it wasn't a hornet."
-				<br>
-				- W.S Gilbert
+	<div class="Section">
+			<div class="Block">
+				<h4 class="Block__Header">Placeholder</h4>
+				<p class="Block__Text">
+					Lorem ipsum dolor sit amet, consectetur adipiscing elit. Fusce risus nibh, gravida nec felis quis, facilisis facilisis lectus.
+					Nulla ac orci pretium, condimentum orci quis, accumsan nisi. Aliquam erat volutpat. Curabitur cursus mattis libero,
+					at viverra risus hendrerit quis. Fusce imperdiet tristique tortor non tincidunt. Mauris accumsan urna nec augue feugiat
+					porta. Proin vitae magna in ex malesuada laoreet eget a nulla. Aliquam tristique et elit at consequat. In hac habitasse
+					platea dictumst.
+				</p>
+			</div>
+			<div class="Block">
+				<h4 class="Block__Header">Placeholder</h4>
+				<p class="Block__Text">
+					Lorem ipsum dolor sit amet, consectetur adipiscing elit. Fusce risus nibh, gravida nec felis quis, facilisis facilisis lectus.
+					Nulla ac orci pretium, condimentum orci quis, accumsan nisi. Aliquam erat volutpat. Curabitur cursus mattis libero,
+					at viverra risus hendrerit quis. Fusce imperdiet tristique tortor non tincidunt. Mauris accumsan urna nec augue feugiat
+					porta. Proin vitae magna in ex malesuada laoreet eget a nulla. Aliquam tristique et elit at consequat. In hac habitasse
+					platea dictumst.
+				</p>
 			</div>
 		</div>
-	</div>
-	<div class="Tabs__item" data-tab="3">
-		<div class="Box">
-			<div class="Box__title">Box # 3</div>
-			<div class="Box__description">A dozen, a gross, and a score
-				Plus three times the square root of four
-				<br>
-				Divided by seven
-				<br>
-				Plus five times eleven
-				<br>
-				Is nine squared and not a bit more.
-				<br>
-				- Leigh Mercer
+		<div class="Section LowerHeader">
+					
```

Comment:
I would suggest that you do an auto-format right before a commit. It will help you with reading through your code a little bit more smoothly! Auto-format does not work with white spaces (extra lines of empty code) so be on the lookout for those!

Reply:
There is another `Header` that I see here and wasn't sure if that was on purpose or not. Either way, your project looks awesome!